npm/cypress-schematic/README.md

```diff
@@ -223,12 +242,22 @@ In order to prevent the application from building, add the following to the end
 
 ## Generator Options
 
+### Specify Testing Type 
+
+The default generated spec is E2E.  In order to generate a component test you can run:
+
+```shell script
+ng generate @cypress/schematic:spec --name=button -t component
+```
+
+`-t` is an alias for `testing-type`. It accepts `e2e` or `component` as arguments. If you are using the CLI tool, a prompt will appear asking which spec type you want to generate.
```

Comment:
not a huge deal but the cypress app uses `--component` & `--e2e` as there flags. What do you think about using that instead of `-t`?

Reply:
Good call. I removed `-t` in favor of a default for `e2e` or  `--component` for generating component tests.